Utils.js

```diff
@@ -13,3 +13,86 @@ export function calcLineItem(quantity, price) {
     return totalLine(Math.round(price * 100) / 100);
 }
 
+export const CART = 'CART';
+
+export function renderGhosts(ghost) {
+    const li = document.createElement('li');
+    li.classList.add(ghost.category);
+    li.title = ghost.description;
+
+    const h3 = document.createElement('h3');
+    h3.textContent = ghost.name;
+    li.appendChild(h3);
+
+    const img = document.createElement('img');
+    img.src = '../assets/' + ghost.image;
+    img.alt = `${ghost.name}image`;
+    li.appendChild(img);
+
+    const p = document.createElement('p');
+    p.classList.add('price');
+    p.textContent = `$${ghost.price.toFixed(2)}`;
+
+    const button = document.createElement('button');
+    const removeButton = document.createElement('button');
+    button.textContent = 'Add to Cart';
+    button.value = ghost.code;
+    p.appendChild(button);
+
+    li.appendChild(p);
+
+
+    button.textContent = 'Add to Cart';
+
+    button.addEventListener('click', () => {
+
+        const cart = getFromLocalStorage(CART) || [];
+        const itemInCart = findById(cart, ghost.id);
+
+        if (itemInCart === undefined) {
+
+            const newCartItem = {
+                id: ghost.id,
+                quantity: 1,
+            };
+            cart.push(newCartItem);
+        } else {
+            itemInCart.quantity++;
+        }
+        setInLocalStorage(CART, cart);
+    });
+    
+    removeButton.textContent = 'remove one';
+
+    removeButton.addEventListener('click', () => {
+        const cart = getFromLocalStorage(CART) || [];
+        const itemInCart = findById(cart, ghost.id);
+
+        if (itemInCart) {
+            if (itemInCart.quantity !== 0) {
+                itemInCart.quantity--;
+            }
+        }
+
+
```

Comment:
the remove button isn't working, you will need to remove it from localstorage. (you might  setInLocalStorage(CART, cart); and play around with that.) (it will cause a bug, but I think you can figure out how to fix it)